ui/src/main/res/layout/layout_xltunai_status.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical" android:layout_width="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:padding="@dimen/sixteen_dp"
+    android:layout_height="match_parent">
+
+    <com.midtrans.sdk.ui.widgets.DefaultTextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="left"
+        android:paddingLeft="@dimen/eight_dp"
+        android:paddingTop="@dimen/sixteen_dp"
+        android:text="@string/text_hint_xl_order_id"
+        android:textColor="@color/dark_gray"
+        android:textSize="@dimen/secondary_text_size" />
+
+    <LinearLayout
+        android:gravity="center_vertical"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+        <com.midtrans.sdk.ui.widgets.DefaultTextView
+            android:id="@+id/text_order_id"
+            android:layout_width="0dp"
+            android:background="@color/light_gray"
+            android:layout_weight="1"
+            android:layout_marginRight="@dimen/four_dp"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:padding="@dimen/eight_dp"
+            android:textColor="@color/black"
+            android:textSize="@dimen/header_text_size"
+            tools:text="@string/text_company_code_here" />
+
+        <com.midtrans.sdk.ui.widgets.FancyButton
```

Comment:
You can use style `buttonSecondary` so you can remove some properties that already defined `fb_borderWidth`, `fb_focusColor`, `fb_defaultColor`, `fb_radius`.

Also you don't need `fb_iconColor`.